demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -34,6 +34,9 @@
 ISO_TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 ALLOWED_CERTIFICATION_VALUES = ['certified', 'verified']
 SUPPORT_TYPES = ['community', 'xsoar'] + SUPPORTED_CONTRIBUTORS_LIST
+INCORRECT_PREFIX_PACK_NAME = ["Playbook", "Integration", "Script"]
+PATTERN = '[^a-zA-Z]pack[^a-z]|^pack$|^pack[^a-z]|[^a-zA-Z]pack$|[^A-Z]PACK[^A-Z]|^PACK$|^PACK[^A-Z]|[^A-Z]PACK$|' \
```

Comment:
I thing we have to be case sensitive because something like this **SomethingPackSomething** have to fail and this **SomethingPackageSomething** should not fail